apache/superset
# fix: replace datamask with key from new key value api

### SUMMARY
This pr fixes the longurl problem in dashboard when a user has a lot of filters and shares the url with other users. The long copied url had the potential to break loading the dashboard. This pr uses the new keyvalue api for dashboard to replace the datamask content with secure hashed key. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

before

https://user-images.githubusercontent.com/17326228/145321730-ad7048c7-cd34-4b8c-b3a6-1d74f595bf22.mov

after

https://user-images.githubusercontent.com/17326228/145322047-cd7cc9d4-7af3-472d-bf9a-3b40cc1cc38a.mov



### TESTING INSTRUCTIONS
Go to a dashboard and create filters. Ensure that the nativefilters param has a hashed key when applying new filters and saving the dashboard url.

### ADDITIONAL INFORMATION
